### PR TITLE
LLMonitor Callback Handler: fix tracking users on agents

### PR DESCRIPTION
**Pull request by vincelwt**
Hello!

This fixes tracking users on agents and chains with the LLMonitor Callback Handler.

Changes tested and linted.

Thank you.

### CODE REVIEW

File: langchain/package.json [hunk 1]
```diff
@@ -812,7 +812,7 @@
     "jest": "^29.5.0",
     "jest-environment-node": "^29.6.4",
     "jsdom": "^22.1.0",
-    "llmonitor": "^0.5.4",
```
Comment (jacoblee93) [inline langchain/package.json:815]: You probably also want to bump the `peerDependency`?

**Comment by jacoblee93** [comment]
Looks fine, it looks like you need to update the lockfile by running `yarn` again though.

File: langchain/src/callbacks/handlers/llmonitor.ts [hunk 2]
```diff
@@ -185,18 +184,17 @@ export class LLMonitorHandler
       ...(metadata || {}),
     };
 
-    const name =
-      params?.model || params?.name || params?.model_name || llm.id.at(-1);
+    // Expend them so they're excluded from the "extra" field
```
Comment (jacoblee93) [inline langchain/src/callbacks/handlers/llmonitor.ts:187]: `Expand`

**Comment by vincelwt** [comment]
@jacoblee93 Done! Thanks

File: langchain/package.json (hunk 1 above)
Comment (jacoblee93) [reply langchain/package.json:836]: Oh sorry, didn't realize it was a `*` already - happy to leave it that way if you want, but will merge this as is.

**Pull request merged**